spec/operators/delay-spec.js

```diff
@@ -1,11 +1,79 @@
 /* globals describe, it, expect, expectObservable, hot, rxTestScheduler */
 var Rx = require('../../dist/cjs/Rx');
+var Observable = Rx.Observable;
 
 describe('Observable.prototype.delay()', function () {
   it('should delay by specified timeframe', function () {
-    var source = hot('--a--|');
-    var expected =   '-----a--|';
+    var e1 =   hot('--a--b--|');
+    var expected = '-----a--b--|';
 
-    expectObservable(source.delay(30, rxTestScheduler)).toBe(expected);
+    expectObservable(e1.delay(30, rxTestScheduler)).toBe(expected);
+  });
+
+  it('should delay by absolute time period', function () {
+    var e1 =   hot('--a--b--|');
+    var expected = '-----a--b--|';
+    var absoluteDelay = new Date(rxTestScheduler.now() + 30);
+
+    expectObservable(e1.delay(absoluteDelay, rxTestScheduler)).toBe(expected);
+  });
+
+  it('should delay by absolute time period after subscription', function () {
+    var e1 =   hot('---^--a--b--|');
+    var expected =    '------a--b--|';
+    var absoluteDelay = new Date(rxTestScheduler.now() + 30);
+
+    expectObservable(e1.delay(absoluteDelay, rxTestScheduler)).toBe(expected);
+  });
+
+  it('should raise error when source raises error', function () {
+    var e1 =   hot('---a---b---#');
+    var expected = '------a---b#';
```

Comment:
`var e1Sub = '^         !';` it was missing `^` to indicate when it's subscribed.